src/execute_files.py

```diff
@@ -32,7 +32,8 @@ def create_outname(outdir, infile):
 def execute(infile, outfile, extras):
   """Create the command-line for an execution."""
   runner = extras['runner']
-  basename = os.path.splitext(os.path.basename(runner))[0]
+  # Only strip Windows suffxes because wasm.opt has one
+  basename = os.path.basename(runner).replace('.exe', '').replace('.bat', '')
```

Comment:
Yeah. After this change (plus my other outstanding ones to finish windows support) I'll probably put some of these into a library, even though the amount of platform-specific stuff ended up being less than I expected.